Helpers/opent2t-translator-helper-hue/js/package.json

```diff
@@ -0,0 +1,12 @@
+{
+    "name": "opent2t-translator-helper-hue",
```

Comment:
I do not think that we need it. Since the module was completed, I thought I would just add it anyway. Any suggest if we should add or remove the hue helper module?